mater/lib/src/stores/mod.rs

```diff
@@ -24,24 +24,38 @@ pub enum Config {
         chunk_size: usize,
         /// The number of children per parent node.
         tree_width: usize,
+        /// If false it's unixfs
+        raw: bool,
     },
 }
 
 impl Config {
     /// Create a new [`Config::Balanced`].
-    pub fn balanced(chunk_size: usize, tree_width: usize) -> Self {
+    pub fn balanced(chunk_size: usize, tree_width: usize, raw: bool) -> Self {
         Self::Balanced {
             chunk_size,
             tree_width,
+            raw,
         }
     }
+
+    /// Creates a new balanced tree configuration with UnixFS wrapping (recommended).
+    pub fn balanced_unixfs(chunk_size: usize, tree_width: usize) -> Self {
+        Self::balanced(chunk_size, tree_width, false)
+    }
+
+    /// Creates a new balanced tree configuration with raw storage.
+    pub fn balanced_raw(chunk_size: usize, tree_width: usize) -> Self {
+        Self::balanced(chunk_size, tree_width, true)
```

Comment:
If VSCode isn't tricking me, these are only used in tests, I think just the `balanced` is enough.